task/buildah-remote-oci-ta/0.1/buildah-remote-oci-ta.yaml

```diff
@@ -214,41 +214,53 @@ spec:
     image: quay.io/redhat-appstudio/multi-platform-runner:01c7670e81d5120347cf0ad13372742489985e5f@sha256:246adeaaba600e207131d63a7f706cffdcdc37d8f600c56187123ec62823ff44
```

Comment:
@ifireball @chmeliik, I just realized there is a remaining issue that we will need to resolve with this PR.
These remote jobs are not run using a buildah image for the container. When we run locally, we will need to either embed our buildah binary into the multi-platform runner or pull a buildah image to run the build in.

Reply:
Ah, buildah is already in it: https://github.com/konflux-ci/build-tasks-dockerfiles/blob/main/multi-platform-runner/Dockerfile but we will need to ensure that this image is up to date since it hasn't changed

Reply:
Can we switch the the image the local task is using? It does not seem like the remote commands require stuff that I wouldn't expect to find in any image

Reply:
The buildah image that we are currently using does not install `rsync`, `openssh-clients` and it does not have `kubectl` either. 

Something will need to be done as I don't think that the runner image is sufficient as it also has an older buildah version than what we currently use (1.29.1 vs 1.38.0-dev).

Reply:
The easiest thing for the short term may be to update the runner to be based on the current buildah image that we use. This will change its parent image from being UBI-based to Fedora based.

Reply:
Resolving this thread now that I have changed to all tasks using https://github.com/konflux-ci/buildah-container/blob/main/Containerfile.task (which is the buildah-task image)